### ISSUE TITLE
Avoid using shared calendar

### ISSUE THREAD

**Issue by hotchemi** [hotchemi/khronos#9]

**Comment by MrTact** [comment]
Yeah, this is interesting -- I almost think you need to do a separate library to make this (and timezones) work, because everything will have to use java.time classes like LocalDate instead of the older java.util classes. Important, though, because of lack of thread safety.

**Comment by hotchemi** [comment]
@MrTact Thx for your interest. Yeah actually I've been thinking the same thing. Are you interested in working on it?

**Comment by MrTact** [comment]
I am! I actually thought about doing something like this, and then realized that someone surely must have done it already.

Then I pulled khronos, thinking "I'll just see if I can add a couple of timezone-related extensions (e.g. "2017-05-10".utc), and then realized that was going to take a lot more work than I anticipated.

**Comment by hotchemi** [comment]
> then realized that someone surely must have done it already.

Oh do you mean it's already fixed?

**Comment by MrTact** [comment]
No, I meant I was thinking about creating a library that used some of Kotlin's cooler features to create a DSL-like wrapper around Java date & time modules. Then I thought someone must have done this already, and someone did -- you :-)

As far as I have been able to find, no one has done a comparable library wrapping LocalDateTime.
